edx_notifications/server/web/static/edx_notifications/js/vendor/dev/mock-ajax.js

```diff
@@ -0,0 +1,576 @@
+/*
+
+Jasmine-Ajax : a set of helpers for testing AJAX requests under the Jasmine
```

Comment:
We are using Sinon in edx-platform for mock AJAX requests. We should probably agree upon a standard library to use in our various repos. Are there advantages of Jasmine-Ajax over Sinon?